assets/queries/ansible/aws/db_security_group_public_scope/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "aws_db_security_group_no_public_scope",
+  "queryName": "AWS DB Security Group with Public Scope",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "The CIDR must be diferent than 0.0.0.0/0",
+  "descriptionUrl": "https://docs.ansible.com/ansible/latest/collections/community/aws/rds_instance_module.html#parameter-db_security_groups"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline